File: Homeworks/Homeworks C#1/OperatorsAndExpressions/07.PointInACircle/PointInACircle.cs
//Problem 7. Point in a Circle

//Write an expression that checks if given point (x, y) is inside a circle K({0, 0}, 2)

using System;
class PointInACircle
{
    static void Main()
    {
        
        double radius = 2.0;
        double x = double.Parse(Console.ReadLine());
        double y = double.Parse(Console.ReadLine());
        bool isInside = (x*x) + (y*y)<= (radius*radius);
        double distance = Math.Sqrt((x * x) + (y * y));
        
        if (isInside==true)
        {
            Console.WriteLine("yes {0:0.00}",distance);
        }
        else
        {
            Console.WriteLine("no {0:0.00}", distance);
        }
    }
}

File: Homeworks/JavaScript Fund/operators&expressions/PointInACircle.js
function PointInACircle(param) {
    let x = +param[0];
    let y = +param[1];
    let R = 2;
    if (x * x + y * y <= 4) {
        console.log('yes' + ' ' + Math.sqrt((x * x) + (y * y)).toFixed(2));

    } else {
        console.log('no' + ' ' + Math.sqrt((x * x) + (y * y)).toFixed(2));

    }


}
PointInACircle(['0.2', '-0.8']);